lib/client.js

```diff
@@ -283,6 +283,7 @@ function isOriginValid (origin) {
       /^https:\/\/.+\.zd-dev\.com$/,
       /^https:\/\/.+\.zd-master\.com$/,
       /^https:\/\/.+\.zendesk-staging\.com$/,
+      /^https:\/\/.+[euc1|usw2]\.scooter\.zendesk-dev\.com$/,
```

Comment:
Sorry, it seems my previous comment make you confusion. I was trying to say domain level(zendesk-dev.com) looks good enough. Not need to specify into subdomain level.

Reply:
Ah ok. I did see the comment from before 😅